index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles2.css">
+    <title>Document</title>
+</head>
+
+
+<body>
+
+<!--header / top part-->>
+    
+  <header class="topbar">
+    <a href="#" class="logo">Welcome</a>
+    <nav><img class="threelinemenu" src="images/Icon-Menu.png" alt="the menu"></nav>
+  </header>
+
+<!--Mid part with basic info-->>
+  <section>
+ 
+    <div class="greeting"> 
+      <h1 class="firstinfo">Hey there. I’m Erick</h1>
+      <h2 class="secondinfo">I’m a software developer</h2>
+    </div>
+
+      
+    <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.</p>
+
+ <!--Social media links at the bottom-->>       
+        
+    <ul class="bottombar">
+      <li><a href="#"><img src="images/twitter.png" alt="twitter logo"></a></li>
```

Comment:
- [ ] Kindly remove those bullet points in every icon in the list.
![Captura de pantalla_2021-09-16_22-19-16](https://user-images.githubusercontent.com/53111464/133719594-8e39bfbf-8712-4279-b966-76b88453a477.png)

Reply:
Thnaks. I did with "list-style-type: none". 
I was not sure if it would have been better to just remove the <li> elements. Advice would be great so I can do it better the next time.

Thanks!

Reply:
Hello @Erick-Osterling 
You can use list-style: none; in order to remove those default bullet points in the list.

Reply:
I was not sure if it would have been better to just remove the elements.
No in further steps you will need those icons as links to your social media profiles.